Quantities notebook in quarantine

As of PR #446, the quantities notebook appears out-of-date enough to [break Circle CI](https://circleci.com/gh/astropy/astropy-tutorials/724?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link).

We need a PR to bring it up to date or move it to the quarantine folder.

It looks like all the notebooks run now in [PR 449](https://github.com/astropy/astropy-tutorials/pull/449) but there is an error in the sphinx build:

```
Exception occurred:
  File "/home/circleci/project/venv/lib/python3.7/site-packages/matplotlib/sphinxext/plot_directive.py", line 262, in setup
    app.add_directive('plot', plot_directive, True, (0, 2, False), **options)
TypeError: add_directive() got an unexpected keyword argument 'alt'
The full traceback has been saved in /tmp/sphinx-err-t2luruki.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make: *** [Makefile:43: html] Error 2

Exited with code exit status 2
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -14,7 +14,7 @@ pip-run: &pip-install
     python3 -m venv venv
     . venv/bin/activate
     pip install -r pip-requirements.txt  --progress-bar off
-    pip install --progress-bar off sphinx>=2.0,<3.0 pytest
+    pip install --progress-bar off sphinx<3.0 pytest
     pip install --progress-bar off sphinx-astropy sphinx-bootstrap-theme nbconvert==5.6.1
 
 jobs:

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -14,7 +14,7 @@ pip-run: &pip-install
     python3 -m venv venv
     . venv/bin/activate
     pip install -r pip-requirements.txt  --progress-bar off
-    pip install --progress-bar off sphinx<3.0 pytest
+    pip install --progress-bar off "sphinx>=2.0,<3.0" pytest
     pip install --progress-bar off sphinx-astropy sphinx-bootstrap-theme nbconvert==5.6.1
 
 jobs:

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -14,7 +14,7 @@ pip-run: &pip-install
     python3 -m venv venv
     . venv/bin/activate
     pip install -r pip-requirements.txt  --progress-bar off
-    pip install --progress-bar off sphinx pytest
+    pip install --progress-bar off sphinx>=2.0,<3.0 pytest
     pip install --progress-bar off sphinx-astropy sphinx-bootstrap-theme nbconvert==5.6.1
 
 jobs:

--- a/tutorials/notebooks/quantities/quantities.ipynb
+++ b/tutorials/notebooks/quantities/quantities.ipynb
@@ -368,7 +368,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "np.log10(M.to(u.Msun).value)"
+    "np.log10(M.to_value(u.Msun))"
    ]
   },
   {

--- a/tutorials/notebooks/quantities/quantities.ipynb
+++ b/tutorials/notebooks/quantities/quantities.ipynb
@@ -368,7 +368,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "np.log10(M / u.Msun)"
+    "np.log10(M.to(u.Msun).value)"
    ]
   },
   {
@@ -1093,11 +1093,6 @@
    "name": "",
    "published": true
   },
-  "kernelspec": {
-   "display_name": "Python 3",
-   "language": "python",
-   "name": "python3"
-  },
   "language_info": {
    "codemirror_mode": {
     "name": "ipython",
@@ -1108,9 +1103,9 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.7"
+   "version": "3.7.6"
   }
  },
  "nbformat": 4,
- "nbformat_minor": 2
+ "nbformat_minor": 4
 }

--- a/tutorials/notebooks/quantities/requirements.txt
+++ b/tutorials/notebooks/quantities/requirements.txt
@@ -1,3 +1,3 @@
 numpy
-matplotlib==3.0.2
+matplotlib==3.1.3
 astropy

